install/boards/bcm_27xx.sh

```diff
@@ -6,16 +6,33 @@ VERSION="${VERSION:-master}"
 GITHUB_REPOSITORY=${GITHUB_REPOSITORY:-bluerobotics/blueos-docker}
 REMOTE="${REMOTE:-https://raw.githubusercontent.com/${GITHUB_REPOSITORY}}"
 ROOT="$REMOTE/$VERSION"
-CMDLINE_FILE=/boot/cmdline.txt
 alias curl="curl --retry 6 --max-time 15 --retry-all-errors"
 
+# Determine OS
+OS=$(lsb_release -is)
+
+if [ "$OS" = "Raspbian" ]; then
+    echo "Running on Raspbian."
+    CMDLINE_FILE="/boot/cmdline.txt"
+    CONFIG_FILE="/boot/config.txt"
+    OVERLAYS_DIR="/boot/overlays"
```

Comment:
@Williangalvani, since you was working on the support to the latest raspbian version, are these paths the same or they changed ?
My point is that such paths can be a consequence of the debian version that such distros are based over (Debian), so the checks should be done with the debian version and not with the distro name. @agregghai any idea ?

Reply:
yes, they are still the same